src/features/student/additionalSkills/components/interactions/inputs/AdditionalSkillCommentInput/AdditionalSkillCommentInput.test.ts

```diff
@@ -0,0 +1,216 @@
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('disabled')).toBe(false)
+    })
+
+    BddTest().then('it should have default required prop set to true', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('required')).toBe(true)
+    })
+
+    BddTest().then('it should have default maxlength prop set to 400', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('maxlength')).toBe(400)
+    })
+
+    BddTest().then('it should use default i18n label', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('label')).toBe('Description de ma compétence')
+    })
+  })
+
+  BddTest().and('the component is mounted with custom props', () => {
+    beforeEach(() => {
+      vi.clearAllMocks()
+      wrapper = mount(AdditionalSkillCommentInput, {
+        props: {
+          isValid: true,
+          isTextarea: false,
+          labelVisible: false,
+          disabled: true,
+          required: false,
+          maxlength: 200,
+          id: 'custom-id',
+          label: 'Custom Label',
+          errorMessage: 'Custom error message'
+        },
+        global: { stubs }
+      })
+    })
+
+    BddTest().then('it should pass custom isValid prop to AvInput', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('isValid')).toBe(true)
+    })
+
+    BddTest().then('it should pass custom isTextarea prop to AvInput', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('isTextarea')).toBe(false)
+    })
+
+    BddTest().then('it should pass custom labelVisible prop to AvInput', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('labelVisible')).toBe(false)
+    })
+
+    BddTest().then('it should pass custom disabled prop to AvInput', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('disabled')).toBe(true)
+    })
+
+    BddTest().then('it should pass custom required prop to AvInput', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('required')).toBe(false)
+    })
+
+    BddTest().then('it should pass custom maxlength prop to AvInput', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('maxlength')).toBe(200)
+    })
+
+    BddTest().then('it should use custom label', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('label')).toBe('Custom Label')
+    })
+
+    BddTest().then('it should pass custom errorMessage to AvInput', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('errorMessage')).toBe('Custom error message')
+    })
+  })
+
+  BddTest().and('the user updates the input value', () => {
```

Comment:
```suggestion
  BddTest().when('the user updates the input value', () => {
```

S'il n'y a pas de when avant un then, alors ce and doit être un when